recipes/xfig/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -x
+
+# Point to the right pkgconfig
+export PKG_CONFIG_LIBDIR="$PREFIX/lib/pkgconfig"
+# Add the build-dir's lib dir to the linker flags
+export LDFLAGS="-L$PREFIX/lib"
+# Add the build-dir's incude dir to the compiler flags
+export CFLAGS="-I$PREFIX/include"
```

Comment:
Same here. I think this to be `export CFLAGS="${CFLAGS} -I$PREFIX/include" so you don't overwrite [what's being set in the toolchain activate.sh](https://github.com/conda-forge/toolchain-feedstock/blob/master/recipe/activate.sh#L34).